cpp/src/cider/tests/nextgen/functional/CMakeLists.txt

```diff
@@ -33,11 +33,7 @@ foreach(next_gen_test_file ${next_gen_test_files})
   string(REGEX MATCH "functional/.*" test_relative_path ${next_gen_test_file})
   string(REGEX REPLACE "functional/" "" target_name ${test_relative_path})
   string(REGEX REPLACE ".cpp" "" target_name ${target_name})
-  if(${target_name} MATCHES "ScalarFunctionExprNextgenTest")
-    continue()
-  elseif(${target_name} MATCHES "SimpleCiderNextgenTest")
```

Comment:
keep and skip these 2 tests `ScalarFunctionExprNextgenTest` and `SimpleCiderNextgenTest` instead of delete.